common/src/main/resources/assets/ftbquests/lang/fr_fr.json

```diff
@@ -0,0 +1,482 @@
+	"ftbquests.gui.save_as_file": "Télécharger les fichiers de quête",
+	"ftbquests.gui.save_on_server": "Enregistrer maintenant sur le serveur",
+	"ftbquests.gui.export_spreadsheet": "Exporter le tableau",
+	"ftbquests.gui.saved_as_file": "Quêtes enregistrées sur le client dans %s",
+	"ftbquests.gui.saved_on_server": "Quêtes enregistrées sur le serveur",
+	"ftbquests.gui.error": "Erreur !",
+	"ftbquests.gui.add_reward_all": "Ajouter des récompenses pour tous...",
+	"ftbquests.gui.clear_reward_all": "Supprimer les récompenses pour tous",
+	"ftbquests.gui.select_quest_object": "Sélectionner un objet de quête",
+	"ftbquests.gui.select_task_type": "Sélectionner un type de tâche",
+	"ftbquests.gui.select_reward_table": "Sélectionner un table de récompenses",
+	"ftbquests.gui.select_reward_type": "Sélectionner un type de récompense",
+	"ftbquests.gui.reset_progress": "Réinitialiser la progression",
+	"ftbquests.gui.reset_progress_q": "Réinitialiser la progression pour votre équipe ?",
+	"ftbquests.gui.complete_instantly": "Compléter instantanément",
+	"ftbquests.gui.complete_instantly_q": "Compléter cette tâche instantanément ?",
+	"ftbquests.gui.add_dependencies": "Ajouter les sélectionnées aux dépendances",
+	"ftbquests.gui.remove_dependencies": "Supprimer les sélectionnées des dépendances",
+	"ftbquests.gui.add_dependencies_self": "Ajouter en tant que dépendances aux sélectionnées",
+	"ftbquests.gui.remove_dependencies_self": "Supprimer en tant que dépendances aux sélectionnées",
+	"ftbquests.gui.copy_id": "Copier l'ID",
+	"ftbquests.gui.copy_id.paste_hint": "Vous pouvez utiliser cette ID pour copier les quêtes ou les liens de quêtes",
+	"ftbquests.gui.copy_id.quick_properties": "Édition rapide des propriétés...",
+	"ftbquests.gui.no_tasks": "Pas de tâches",
+	"ftbquests.gui.no_rewards": "pas de récompenses",
+	"ftbquests.gui.requires": "À besoin de",
+	"ftbquests.gui.required_by": "Requis par",
+	"ftbquests.gui.collect_rewards": "Clic pour collecter toutes les récompenses !",
+	"ftbquests.gui.times_completed": "Temps accomplis",
+	"ftbquests.gui.looping_dependencies": "Boucle les dépendances !",
+	"ftbquests.gui.open_in_guide": "Cliquez ici pour lire plus...",
+	"ftbquests.gui.wiki": "Ouvrir le Wiki",
+	"ftbquests.gui.change_progress": "Changer la progression",
+	"ftbquests.gui.change_progress.reset": "Réinitialiser",
+	"ftbquests.gui.change_progress.reset_deps": "Réinitialiser (+Dépendances)",
+	"ftbquests.gui.change_progress.complete": "Compléter",
+	"ftbquests.gui.change_progress.complete_deps": "Compléter (+Dépendances)",
+	"ftbquests.gui.pin": "Épingler",
+	"ftbquests.gui.unpin": "Désépingler",
+	"ftbquests.gui.autopin.on": "Épingle automatique: Activée",
+	"ftbquests.gui.autopin.off": "Épingle automatique: Désactivée",
+	"ftbquests.gui.move_to_chapter": "Aller à un autre chapitre",
+	"ftbquests.gui.change_group": "Changer de groupe",
+	"ftbquests.gui.move_tooltip": "Raccourci: Clic molette de la souris",

```

Comment:
In French there is a space between the word "raccourci" and the punctuation mark ":"